pkgs/development/libraries/wavpack/default.nix

```diff
@@ -1,21 +1,22 @@
-{ stdenv, fetchurl, libiconv }:
+{ lib, stdenv, fetchurl, libiconv }:
 
 stdenv.mkDerivation rec {
   pname = "wavpack";
-  version = "5.3.0";
+  version = "5.4.0";
 
   enableParallelBuilding = true;
 
   buildInputs = stdenv.lib.optional stdenv.isDarwin libiconv;
 
   src = fetchurl {
-    url = "http://www.wavpack.com/${pname}-${version}.tar.bz2";
-    sha256 = "00baiag7rlkzc6545dqdp4p5sr7xc3n97n7qdkgx58c544x0pw5n";
+    url = "https://www.wavpack.com/${pname}-${version}.tar.bz2";
+    sha256 = "0d60r8bfvnm6j8ckajjfrcvrix5ac6yvn28vbq062bd7z7fac5h7";
   };
 
-  meta = with stdenv.lib; {
+  meta = with lib; {
     description = "Hybrid audio compression format";
-    homepage    = "http://www.wavpack.com/";
+    homepage    = "https://www.wavpack.com/";
+    changelog   = "https://github.com/dbry/WavPack/releases/tag/${version}";
```

Comment:
TIL.

Reply:
The NEWS file might actually be a better candidate for this, as it seems to include more information.

Reply:
I like the GitHub page better since it links to other information in addition to the NEWS file.

Reply:
fair enough